.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+## How to contribute
+
+Share your knowledge with the community by contributing to Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [devdocs](https://GitHub.com/magento/devdocs){:target="_blank"} GitHub repository.
```

Comment:
GH markdown does not support custom attributes so we should remove `{:target="_blank"}` everywhere on this page.

Reply:
Removing!